.github/workflows/macos.yml

```diff
@@ -74,10 +82,16 @@ jobs:
           repository: ${{ inputs.sage_repo }}
           ref: ${{ inputs.sage_ref }}
           fetch-depth: 10000
-
+      - uses: actions/setup-python@v5
+        # As of 2024-02-03, the macOS M1 runners do not have preinstalled python or pipx.
+        # Installing pipx follows the approach of https://github.com/pypa/cibuildwheel/pull/1743
+        id: python
+        with:
+          python-version: "3.8 - 3.12"
+          update-environment: false
```

Comment:
why do you set update-environment to false? Otherwise you could just use `python` below.

Reply:
Because i want to use this Python only for running tox, not for influencing the Sage build.

Reply:
If it would have a preinstalled python, it would also influence the sage build.